File: src/main/java/ru/citeck/idea/codeinsight/forms/components/SelectJournal.java
package ru.citeck.idea.codeinsight.forms.components;

import lombok.Getter;

@Getter
public class SelectJournal extends AssocComponent {

    public static final String TYPE = "selectJournal";

    private final String modalTitle = "";
    private final String journalId = "";
    private final String queryData = "";


    @Override
    public String getType() {
        return TYPE;
    }

}

File: src/main/kotlin/ru/citeck/idea/artifacts/action/codeinsight/forms/components/SelectJournal.kt
package ru.citeck.idea.artifacts.action.codeinsight.forms.components

class SelectJournal : AssocComponent() {

    companion object {
        const val TYPE = "selectJournal"
    }

    val modalTitle = ""
    val journalId = ""
    val queryData = ""

    override fun getType(): String {
        return TYPE
    }
}
